src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -320,6 +320,7 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     if (this.table && this.table.element.clientWidth !== this.currentWidth) {
       this.currentWidth = this.table.element.clientWidth;
       this.table.recalculate();
+      window.dispatchEvent(new Event('resize'));
```

Comment:
It comes out that this solution has some side-effects. A better solution is to notify Angulars change detection: https://github.com/ceph/ceph/pull/30335/files.